### PR TITLE
Ticket_1533

### PR DESCRIPTION
**Pull request by arianalaur3n**
Modifies the Connection Priority column in the Event Channel table.

### CODE REVIEW

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx [hunk 1]
```diff
@@ -142,7 +143,8 @@ const MeterEventChannelWindow = (props: IProps) => {
         });
     }
 
-    
+    console.log(assets);
+    console.log(data);
```
Comment (elwills) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx:147]: Please remove all `cosole.log` statements.

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx [hunk 2]
```diff
@@ -190,6 +192,8 @@ const MeterEventChannelWindow = (props: IProps) => {
         return true;
     }
 
+    
+
```
Comment (elwills) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx:196]: Please remove unnecessary added spaces.

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx (hunk 1 above)
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx:147]:
```suggestion
```

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx (hunk 2 above)
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx:196]:
```suggestion
```

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ProcessedFile/ByFile.tsx [hunk 3]
```diff
@@ -159,7 +159,7 @@ const ByFile: Application.Types.iByComponent = (props) => {
                 />
             </div>
 
-            <Modal Show={selectedID != null} Title={'Events'} CallBack={(c,b) => {
+            <Modal Show={selectedID != null} Title={selectedID?.FilePath ?? 'PathName'} CallBack={(c,b) => {
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ProcessedFile/ByFile.tsx:162]:
```suggestion
            <Modal Show={selectedID != null} Title={'Events'} CallBack={(c,b) => {
```

**Review by clackner-gpa — CHANGES_REQUESTED** [review]

File: Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx [hunk 4]
```diff
@@ -35,6 +35,7 @@ import { SelectAscending, SelectSortKey, SelectEventChannels, SelectEventChannel
 import { FetchChannels } from '../Store/EventChannelSlice';
 import { IsNumber } from '@gpa-gemstone/helper-functions';
 import { cloneDeep } from 'lodash';
+import { getAsset } from '../../../TS/Services/Asset';
```
Comment (clackner-gpa) [inline Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/MeterEventChannel.tsx:38]:
```suggestion
```

**Review by clackner-gpa — APPROVED** [review]

**Pull request merged**